unblob/handlers/filesystem/cramfs.py

```diff
@@ -8,9 +9,14 @@
 from ...file_utils import Endian, convert_int32, get_endian
 from ...models import File, HexString, StructHandler, ValidChunk
 
+CRAMFS_FLAG_FSID_VERSION_2 = 0x00000001
 BIG_ENDIAN_MAGIC = 0x28_CD_3D_45
 
 
+def swap32(i):
```

Comment:
`swap_int32` may be a name that better aligns with the other stuff we have in file_utils

Reply:
sounds good, done.